### PR TITLE
Matthew print and share

### PR DESCRIPTION
**Pull request by MatthewHiscock**
Tweaks and some culling as per @jackparmer's request. @doeg @NicoleGrondin @chriddyp

### CODE REVIEW

File: _posts/2015-04-23-printing-and-sharing-graphs.md [hunk 1]
```diff
@@ -21,24 +21,18 @@ At Plotly, we love sharing. It’s easy to share or print a Plotly chart. Here
...
+To share from the Organize tab, click the Share button as indicated. | ![Share button](/static/images/printing-and-sharing-graphs/share-button.png)
 
-In your workspace, look for the Download button in the toolbar above your graph. To save time, you can use our keyboard shortcuts. On a Mac, “Command-P” and on a PC “Control-P” | ![How to share and print plotly graphs](/static/images/printing-and-sharing-graphs/image02.png)
+In the **Share Plot** popover you can set your graph to be public or private. You can read more about what this means on our <a href="http://help.plot.ly/how-sharing-works-in-plotly/">How Sharing Works</a> page. The **Collaborators** page lets you share with specific email addresses while the **Sharable Link** page provides you with either a private or public URL. Users you assign as collaborators can edit your graphs with you. You can do this with both public and private graphs. | ![Collaborators page](/static/images/printing-and-sharing-graphs/collaborators.png) | ![Sharable link](/static/images/printing-and-sharing-graphs/sharable-link.png)
```
Comment (doeg) [inline _posts/2015-04-23-printing-and-sharing-graphs.md]: Collaborators can be added via both email and username, I think. @alexander-daniel can you confirm?

**Comment by doeg** [comment]
:dancer: Looks good to me, apart from the one thing about collaborators. You'll probably want to get another :dancer: from someone though, since I'm not super familiar with the static pages stuff.

**Comment by MatthewHiscock** [comment]
Great - thanks @doeg ! I'll hold for comment from @alexander-daniel on the collaborators stuff. Who's a good person to ask about the static pages stuff?

**Comment by doeg** [comment]
@MatthewHiscock @NicoleGrondin :)

File: _posts/2015-04-23-printing-and-sharing-graphs.md (hunk 1 above)
Comment (alexander-daniel) [inline _posts/2015-04-23-printing-and-sharing-graphs.md]: Yup, can confirm. But they need to have an account already, you can't just add an e-mail without it being associated to a username already.

File: _posts/2015-04-23-printing-and-sharing-graphs.md [hunk 2]
```diff
@@ -21,24 +21,18 @@ At Plotly, we love sharing. It’s easy to share or print a Plotly chart. Here
...
-The “Sharing settings” box gives you complete control of your chart&#8217;s privacy settings. You can decide if you want your graph to be public or private. To share your graph on a public site, you’ll need to make sure you select Public. Plotly allows you to share your graph and data with collaborators as well. Users you assign as collaborators can edit your graphs with you. You can do this with both public and private graphs. Student options can be selected to help teachers sort their students’ work quickly. | ![How to share and print plotly graphs](/static/images/printing-and-sharing-graphs/image07.png)
+Sharing options can be found in the lower right hand corner of a plot. You can use a direct link, a code to embed the graph in your website or blog (we have an <a href="https://plot.ly/embed/">tutorial on embedding</a> for that!), or share using Facebook, Twitter or Google+. | ![Sharing options](/static/images/printing-and-sharing-graphs/sharing-options.png)
```
Comment (NicoleGrondin) [inline _posts/2015-04-23-printing-and-sharing-graphs.md]: Do we want to specify there the sharing options are located? it tends to change quite a lot depending from where the graphs are viewed...

Comment (MatthewHiscock) [inline _posts/2015-04-23-printing-and-sharing-graphs.md]: @NicoleGrondin Good call. I expanded the Sharing options. Is that what you were thinking, or have I missed anything?

**Comment by NicoleGrondin** [comment]
:dancers: fabulous!

**Pull request merged**